django/src/rdwatch/views/model_run.py

```diff
@@ -165,6 +167,9 @@ def create(self, request: Request):
             title=write_serializer.validated_data['title'],
             performer=write_serializer.validated_data['performer'],
             parameters=write_serializer.validated_data['parameters'],
+            expiration_time=timedelta(
+                hours=write_serializer.validated_data['expiration_time']
+            ),
```

Comment:
In this same file, it may be a good idea to include the expiration_time in the /model-runs endpoint so the client can display when the model-run expires/deletes.

Reply:
:+1: https://github.com/ResonantGeoData/RD-WATCH/pull/223/commits/568a891d822ff97bf2d02c9945d6cdd8b6252374